tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -671,15 +671,18 @@ final case class TestingServer(
   // note(@tgodzik) all test should have `System.exit(0)` added to avoid occasional issue due to:
   // https://stackoverflow.com/questions/2225737/error-jdwp-unable-to-get-jni-1-2-environment
   private def assertSystemExit(parameter: AnyRef) = {
-    def check() = {
+    def check() = try {
+      val nonTarget = workspace.list.filter(_.filename != "target")
```

Comment:
are you sure `target` will be top-level?

Reply:
It was here: `e2e/sbt-debug-step/step-into-scala-lib-virtualdoc/target/scala-2.13/test-classes.bak`